plone/app/portlets/browser/manage.py

```diff
@@ -391,6 +397,8 @@ def __init__(self, context, request):
         # Disable the left and right columns
         self.request.set("disable_plone.leftcolumn", 1)
         self.request.set("disable_plone.rightcolumn", 1)
+        # Disable the ajax main template for special response handling.
+        self.request.set("ajax_load", False)
```

Comment:
This one should not be needed, as this is already done in the `super` call.

Reply:
Thanks, that's right. I removed that one.